python/Ganga/GPIDev/Adapters/IPrepareApp.py

```diff
@@ -45,7 +47,8 @@ def prepare(self, force=False):
         """
         Base class for all applications which can be placed into a prepared\
         state. 
-
+        Args:
+            force (bool) : Fforces the prepare function to be called no matter what when True
```

Comment:
This is going to take another 20min for the sake of a typo. I'll merge the PR and edit it on develop